packages/core/src/routes/sign-in-experience.ts

```diff
@@ -30,12 +30,19 @@ export default function signInExperiencesRoutes<T extends AuthedRouter>(router:
   router.patch(
     '/sign-in-exp',
     koaGuard({
-      body: SignInExperiences.createGuard.omit({ id: true }).partial(),
+      body: SignInExperiences.createGuard
+        .omit({ id: true, termsOfUseUrl: true })
+        .merge(
+          object({
+            termsOfUseUrl: string().url().optional().or(literal('')),
+          })
```

Comment:
Feels like the previous implementation are more clear.  Is there any specific reason to put it in koaGuard?